tests/publiccloud/azure_aitl.pm

```diff
@@ -0,0 +1,129 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in Azure using azure-cli binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use mmapi 'get_current_job_id';
+use utils qw(zypper_call);
+use JSON;
+use XML::LibXML;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    assert_script_run('az version');
+
+    my $provider = $self->provider_factory();
+
+    my $region = get_var('PUBLIC_CLOUD_REGION', 'westus1');
+    my $resource_group = "openqa-aitl-$job_id";
+    my $subscription_id = $provider->provider_client->subscription;
+
+    my $aitl_client_version = "20241118.1";
+    my $aitl_image_gallery = "test_image_gallery";
+    my $aitl_image_version = "latest";
+    my $aitl_job_name = "openqa-aitl-$job_id";
+    my $aitl_manifest = "shared_gallery.json";
+    my $aitl_image_name = $provider->generate_azure_image_definition();
+
+    my $aitl_get_options = "-s $subscription_id -r $resource_group -n $aitl_job_name";
+
+    my $openqa_url = get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
+    my $created_by = "$openqa_url/t$job_id";
+    my $tags = "openqa-aitl=$job_id openqa_created_by=$created_by openqa_var_server=$openqa_url";
+
+    # Get the AITL script
+    assert_script_run("curl https://raw.githubusercontent.com/microsoft/lisa/refs/tags/$aitl_client_version/microsoft/utils/aitl/aitl.py -o /tmp/aitl.py");
+
+    # Configure default location and create Resource group
+    assert_script_run("az configure --defaults location=$region");
```

Comment:
can you elaborate how this line correlate with this section in scenaion JSON ? 
```
           "region": [
                "westus",
                "westus2",
                "westus3"
            ],
```

so which region(s) will be used after all ?

Reply:
TL;DR; Curently the AITL Job Template resource or whatever its called in Azure is available in the WestUS region only (from what I've seen). 

So the AITL Jobs have to be scheduled in that region, however from testing locally I have seen that the resource groups and other objects can be created wherever.